examples/4_point_correlator_full.jl

```diff
@@ -0,0 +1,20 @@
+using ITensors, ITensorMPS
+using ITensorCorrelators
+
+function main(N)
+  sites = siteinds("Electron", N)
+  psi = random_mps(sites, 20) + im * random_mps(sites, 20)
+
+  # get all possible sets of indices
+  indices = vec([tuple(i, j, k, l) for i in 1:N, j in 1:N, k in 1:N, l in 1:N])
+  # using correlators
+  C = correlator(psi, ("Cdagup", "Cdagdn", "Cdn", "Cup"), indices)
+  # computing old-fashioned way
+  for idx in indices
+    ampo = OpSum()
+    ampo += "Cdagup", idx[1], "Cdagdn", idx[2], "Cdn", idx[3], "Cup", idx[4]
+    mpo = MPO(ampo, sites)
```

Comment:
```suggestion
    o = OpSum()
    o += "Cdagup", idx[1], "Cdagdn", idx[2], "Cdn", idx[3], "Cup", idx[4]
    mpo = MPO(o, sites)
```
We are trying to move people away from the `ampo`/`AutoMPO` terminology.